watertap/unit_models/tests/test_unit_test_harness.py

```diff
@@ -0,0 +1,97 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2023, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pytest
+from pyomo.environ import ConcreteModel
+from idaes.core import FlowsheetBlock
+from watertap.unit_models.reverse_osmosis_0D import (
+    ReverseOsmosis0D,
+    ConcentrationPolarizationType,
+    MassTransferCoefficient,
+    PressureChangeType,
+)
+import watertap.property_models.NaCl_prop_pack as props
+from watertap.unit_models.tests.unit_test_harness import UnitTestHarness
+
+from idaes.core.solvers import get_solver
+from idaes.core.util.scaling import calculate_scaling_factors
+
+# -----------------------------------------------------------------------------
+# Get default solver for testing
+solver = get_solver(options={"bound_push": 1e-8})
+
+
+# -----------------------------------------------------------------------------
+class TestReverseOsmosis0D_default(UnitTestHarness):
+    def configure(self):
+        self.model = ConcreteModel()
+        # build unit
+        m = ConcreteModel()
```

Comment:
I believe that this would cause `self.model` and `m` to refer to two completely separate instances.

Python actually lets you assign multiple names (left-hand-side) to the same object (right-hand-side) in one statement, which I frequently use for these cases:

```suggestion
        m = self.model = ConcreteModel()
```